### DISCUSSION TITLE
How can I move radiobuttons away from each other?

### DISCUSSION THREAD

**Discussion by dvd1221** [flet-dev/flet#1128] [General]
Like this
![image](https://user-images.githubusercontent.com/126869058/222733583-9be05197-fa6c-4e85-b15e-9a81445fa73d.png)

**Comment by iron3oxide** [comment]
These kind of questions should be asked in the discussions as they don't relate to a bug or feature request regarding Flet. Please close this issue and open a discussion instead.

**Comment by ndonkoHenri** [comment]
If they are in a Column, then make use of the [`.spacing`](https://flet.dev/docs/controls/column#spacing) property or the [`.alignment`](https://flet.dev/docs/controls/column#alignment)

**Reply by dvd1221** [reply]
Ye, thanks!
